### ISSUE TITLE
error: ‘module_mutex’ undeclared (first use in this function); did you mean ‘module_put’?

### ISSUE THREAD

**Issue by kagaroth** [nbulischeck/tyton#35]
Hello,

There is an issue with the latest version of tyton code. When trying to compile it I get the error in the subject.

OS: Blackarch Linux
Kernel version: 5.13.13-arch1-1
Tyton version: master brach from github and 1:80.56494f3-1 from Blackarch repository.
 ```
DKMS make.log for tyton-80.56494f3 for kernel 5.13.13-arch1-1 (x86_64)
sâmbătă 28 august 2021, 17:11:15 +0300
make -C /lib/modules/5.13.13-arch1-1/build M=/var/lib/dkms/tyton/80.56494f3/build modules
make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
make[1]: Entering directory '/usr/lib/modules/5.13.13-arch1-1/build'
  CC [M]  /var/lib/dkms/tyton/80.56494f3/build/src/core.o
  CC [M]  /var/lib/dkms/tyton/80.56494f3/build/src/util.o
  CC [M]  /var/lib/dkms/tyton/80.56494f3/build/src/proc.o
/var/lib/dkms/tyton/80.56494f3/build/src/proc.c: In function ‘analyze_fops’:
/var/lib/dkms/tyton/80.56494f3/build/src/proc.c:116:29: error: ‘module_mutex’ undeclared (first use in this function); did you mean ‘module_put’?
  116 |                 mutex_lock(&module_mutex);
      |                             ^~~~~~~~~~~~
      |                             module_put
/var/lib/dkms/tyton/80.56494f3/build/src/proc.c:116:29: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [scripts/Makefile.build:273: /var/lib/dkms/tyton/80.56494f3/build/src/proc.o] Error 1
make[1]: *** [Makefile:1862: /var/lib/dkms/tyton/80.56494f3/build] Error 2
make[1]: Leaving directory '/usr/lib/modules/5.13.13-arch1-1/build'
make: *** [Makefile:19: module] Error 2
```
If I run **make** to compile the code I have downloaded from git I get the exact same error. This happens each time I try to compile that module. Tried with different kernel versions, 2 or 3 versions, and always the compilation ends in error.
